server/metrics/metrics.go

```diff
@@ -155,6 +155,15 @@ func NewMetrics(info InstanceInfo) Metrics {
 	})
 	m.registry.MustRegister(m.httpErrorsTotal)
 
+	m.changeEventRejectedTotal = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace:   MetricsNamespace,
+		Subsystem:   MetricsSubsystemEvents,
+		Name:        "change_event_rejected_total",
+		Help:        "The total number of change events rejected due to the activity queue size being full.",
+		ConstLabels: additionalLabels,
+	})
+	m.registry.MustRegister(m.changeEventRejectedTotal)
+
```

Comment:
Propose we name this as `change_event_queue_rejected_total` to correspond with the existing `change_event_queue_capacity` and `change_event_queue_length`.